versions.yaml

```diff
@@ -14,21 +14,36 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-
-_variables:
-  from9: &from9 gcr.io/google-appengine/debian9:latest
 versions:
+  - dir: '2/debian9/2.4'
+    repo: haproxy2
+    tags:
+      - '2.4.2-debian9'
+      - '2.4-debian9'
+      - '2.4.2'
+      - '2.4'
+      - '2-debian9'
+      - '2'
+      - 'latest'
+    excludeTests:
+      - tests/functional_tests/version_2.0_test.yaml
+    from: marketplace.gcr.io/google/debian10
+    templateSubDir: debian10
+    packages:
+      haproxy:
+        sha256: edf9788f7f3411498e3d7b21777036b4dc14183e95c8e2ce7577baa0ea4ea2aa
+        version: '2.4.2'
   - dir: '2/debian9/2.0'
     repo: haproxy2
     tags:
       - '2.0.14-debian9'
       - '2.0-debian9'
       - '2.0.14'
       - '2.0'
-      - '2-debian9'
-      - '2'
-      - 'latest'
-    from: *from9
+    excludeTests:
+      - tests/functional_tests/version_test.yaml
```

Comment:
Instead of disabling this test, can we modify the test to accept both? It looks like it's regex so I'm assuming something like this? '^HA-?Proxy version'

Reply:
@gibbleyg Could you check it now? I've created a regexp)

Reply:
If the test is accepting regex, you should remove exclude entry